src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -44,121 +61,319 @@ export function formatDate(date) {
-      .map((issue, i) => (<li key={`allowed-${i}`}>{issue.description}</li>));
-    const deniedIssues = decisionIssues
-      .filter((issue) => (issue.disposition === 'denied'))
-      .map((issue, i) => (<li key={`denied-${i}`}>{issue.description}</li>));
-    const businessDays = details.businessDays;
-    contents.title = 'The Board has made a decision on some of your appeals';
-    contents.description = (
-      <div>
+  switch (statusType) {
+    case STATUS_TYPES.pendingSoc:
+      contents.title = 'A Decision Review Officer is reviewing your appeal';
+      contents.description = (
+        <p>The Veterans Benefits Administration received your Notice of Disagreement. A Decision
+        Review Officer will review all of the evidence related to your appeal, including any new
+        evidence you sent. The officer may contact you to request additional evidence or medical
+        examinations, as needed. When the officer has completed their review, they will determine
+        whether or not they can grant your appeal.</p>
+      );
+      break;
+    case STATUS_TYPES.pendingForm9:
+      contents.title = 'Please review your Statement of the Case';
+      contents.description = (
         <div>
-          The Board of Veterans Appeals has made a decision on some of your appeals.
-          You will receive your decision letter in the mail in {businessDays} business days. Here is an overview
-          of the decision:
-        </div>
-        <br/>
-        <div className="decision-items">
-          <h5 className="allowed-items">Allowed</h5>
-          <ul>{allowedIssues}</ul>
-          <h5 className="denied-items">Denied</h5>
-          <ul>{deniedIssues}</ul>
+          <p>
+            The Veterans Benefits Administration sent you a Statement of the Case on [date]. The
+            Statement of the Case explains the reasons why they could not fully grant your appeal.
+          </p>
+          <p>
+            If you don’t agree with the Statement of the Case, you can bring your appeal to the Board
+            of Veterans’ Appeals. To do this you must complete and return a Form 9 within 60 days.
+          </p>
         </div>
+      );
+      break;
+    case STATUS_TYPES.pendingCertification:
+      contents.title = 'The Decision Review Officer is finishing their review of your appeal';
+      contents.description = (
+        <p>The Veterans Benefits Administration received your Form 9 and will transfer your appeal
+        to the Board of Veterans’ Appeals. Before doing so, the Decision Review Officer must
+        certify that they have finished reviewing all of the evidence related to your appeal.</p>
+      );
+      break;
+    case STATUS_TYPES.pendingCertificationSsoc:
+      contents.title = 'Please review your new Statement of the Case';
+      contents.description = (
         <div>
-          For issues that are allowed, you will receive compensation. For more information, please
```

Comment:
Change 'For issues that are allowed, you will receive compensation. For more information, please contact your VSO or representative.' to 'If this decision changes your disability rating or your eligibility for VA benefits, you should expect this adjustment to be made in 1 to 2 months.'